libs/pages/database/src/lib/ui/container/container.tsx

```diff
@@ -22,6 +23,7 @@ export interface ContainerProps {
 }
 
 export function Container(props: PropsWithChildren<ContainerProps>) {
+  // TODO remove database.status
```

Comment:
not needed ? The database status is already below?

Reply:
fixed